src/component/auction-page/auction-page.component.jsx

```diff
@@ -35,7 +37,11 @@ export const AuctionPage = () => {
           <S.CurrentPrice>Current Price: $140</S.CurrentPrice>
         </S.WhiteContainer>
         <S.WhiteContainer>
-          <S.TextIfLoggedOut><S.StyledLink>Register</S.StyledLink> or <S.StyledLink>login</S.StyledLink> to place a bid and to participate on the auction.</S.TextIfLoggedOut>
+          <S.TextIfLoggedOut>
+            <S.StyledLink>Register</S.StyledLink> or{" "}
+            <S.StyledLink>login</S.StyledLink> to place a bid and to participate
+            on the auction.
+          </S.TextIfLoggedOut>
         </S.WhiteContainer>
         {!isLoggedIn && <S.RegisterButton>Register </S.RegisterButton>}
         {!isLoggedIn && <S.LogInButton>Log In</S.LogInButton>}
```

Comment:
No need to check isLoggedIn twice, you can replace this with 
{ isLoggedIn $$ (
  <>
        <S.RegisterButton>Register </S.RegisterButton>
        <S.LogInButton>Log In</S.LogInButton>
  </>)}